docs/tutorials/getting-started.md

```diff
@@ -39,7 +39,7 @@ Monitor the deployment:
 
 List OSDs and available unpartitioned disks on each node:
```

Comment:
I think it is okay to have all the OSDs in the tutorial. The action parameters get documented via the actions.yaml so we don't need to make doc changes here.

Reply:
Fixed in subsequent push